libs/src/services/solana/tokenAccount.ts

```diff
@@ -54,7 +53,7 @@ export async function getTokenAccountInfo({
     connection,
     mintKey,
     solanaTokenProgramKey,
-    Keypair.generate()
+    randomKeyPair
```

Comment:
This is the only one of these that's actively used. All the anchor stuff can be deleted. This should be safe 
```
export class Token {
    publicKey: PublicKey;
    programId: PublicKey;
    associatedProgramId: PublicKey;
    payer: Signer;
    constructor(
      connection: Connection,
      publicKey: PublicKey,
      programId: PublicKey,
      payer: Signer,
    );
```

because this is a dummy payer

Reply:
clutch!